.travis.yml

```diff
@@ -5,48 +5,42 @@ matix:
   - python: 3.6
     env:
       - TOX_ENV=py36-flake8
-  - python: 3.4
-    env:
-      - TOX_ENV=py34-django20
-  - python: 3.5
-    env:
-      - TOX_ENV=py35-django20
   - python: 3.6
     env:
-      - TOX_ENV=py36-django20
+      - TOX_ENV=py36-django22
   - python: 3.7
     env:
-      - TOX_ENV=py37-django20
-  - python: 3.5
-    env:
-      - TOX_ENV=py35-django21
-  - python: 3.6
-    env:
-      - TOX_ENV=py36-django21
-  - python: 3.7
+      - TOX_ENV=py37-django22
+  - python: 3.8
     env:
-      - TOX_ENV=py37-django21
-  - python: 3.5
+      - TOX_ENV=py38-django22
+  - python: 3.9
     env:
-      - TOX_ENV=py35-django22
+      - TOX_ENV=py39-django22
   - python: 3.6
     env:
-      - TOX_ENV=py36-django22
+      - TOX_ENV=py36-django31
   - python: 3.7
     env:
-      - TOX_ENV=py37-django22
+      - TOX_ENV=py37-django31
   - python: 3.8
     env:
-      - TOX_ENV=py38-django22
+      - TOX_ENV=py38-django31
+  - python: 3.9
+    env:
+      - TOX_ENV=py39-django31
   - python: 3.6
     env:
-      - TOX_ENV=py36-django30
+      - TOX_ENV=py36-django32
   - python: 3.7
     env:
-      - TOX_ENV=py37-django30
+      - TOX_ENV=py37-django32
   - python: 3.8
     env:
-      - TOX_ENV=py38-django30
+      - TOX_ENV=py38-django32
+  - python: 3.9
+    env:
+      - TOX_ENV=py39-django32
```

Comment:
To avoid manually specifying all versions, we can try: https://github.com/tox-dev/tox-travis#usage

Reply:
Note this should be installed only on `.travis.yml`, so you can add it to the `install` block below

Reply:
@fjsj Thanks for the tip, it worked